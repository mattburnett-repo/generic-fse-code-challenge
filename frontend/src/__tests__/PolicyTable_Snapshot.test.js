
// https://github.com/testing-library/jest-dom

import {renderer, MockedProvider, Policies, mockOperation} from '../test/polityTestConfig'
import 'jest-canvas-mock';

describe('PolicyTable snapshot', () => {
    it('creates a snapshot', () => {
        const tree = renderer.create(
            <MockedProvider mocks={[mockOperation]} addTypename={false}>
                <Policies />
            </MockedProvider>
        )
        expect(tree).toMatchSnapshot();  
    });
}) // end PolicyTable snapshot

