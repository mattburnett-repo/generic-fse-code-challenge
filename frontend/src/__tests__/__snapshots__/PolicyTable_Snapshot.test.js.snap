// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PolicyTable snapshot creates a snapshot 1`] = `
Array [
  <div
    className="spinner"
  />,
  <div
    aria-label="loading-text"
    className="spinnerText"
    role="presentation"
  >
    ... loading
  </div>,
]
`;
